src/index.html

```diff
@@ -11,7 +11,176 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+      class="form"
+      id="out-form"
+      >
+      <fieldset class="form-field">
+        <legend>Personal information</legend>
+        <div class="input-field">
+          <label for="surname">Surname:
+            <input type="text"
+              name="surname"
+              id="surname"
+              pattern="\w+"
+              required
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="input-field">
+          <label for="name">Name:
+            <input type="text"
+              name="name"
+              id="name"
+              pattern="\w+"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+        <div class="input-field">
+          <label for="age">How old are You?
+            <input type="number"
+              name="age"
+              id="age"
+              value="12"
+              pattern="\d+"
+              min = "1"
+              max = "100"
+            >
+          </label>
+        </div>
+        <div class="input-field">
+          <label for="date-of-birth">Full date of birth:
+            <input
+              type="date"
+              name="date"
+              id="date"
+            >
+          </label>
+        </div>
+        <div class="input-field">
+          <label for="accept">I accept the term of the agreement
+            <input type="checkbox">
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="form-field">
+        <legend>Registration</legend>
+        <div>
```

Comment:
Some divs are missing a proper class, that is why we see problems with margins :)